cursive-core/Cargo.toml

```diff
@@ -64,13 +65,40 @@ default-features = false
 optional = true
 version = "0.9"
 
+[dependencies.js-sys]
+version = "0.3.64"
+optional = true
+
+[dependencies.getrandom]
+optional = true
+version = "0.2"
+features = ["js"] 
+
+
+[dependencies.web-sys]
+optional = true
+version = "0.3.64"
+features = [
+    "Window",
+]
+
+[dependencies.wasm-bindgen]
+optional = true
+version = "0.2.87"
+
+[dependencies.wasm-bindgen-futures]
+optional = true
+version = "0.4.37"
+
 [features]
-default = []
+default = ["wasm"]
```

Comment:
Also, this shouldn't be de default - again breaks the build due to selecting async-features (which are fundamentally incompatible with the other backends).